### PR TITLE
Squashed window derivative functions in RasterMonochrome to getFirstDerivative~window

### PR DESCRIPTION
**Pull request by tomasmahlberg**
Overloading this function in #884, hence the `~window`.

### CODE REVIEW

**Comment by tomasmahlberg** [comment]
@emilwestergren Can you take a quick look at this?

File: source/draw/RasterMonochrome.ooc [hunk 1]
```diff
@@ -171,9 +171,8 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		result
 	}
-
 	// get the derivative on small window, region is window's global location on image, window is left top centered.
-	getFirstDerivativeWindowOptimized: func (region: IntBox2D, imageX, imageY: FloatImage) {
+	getFirstDerivative: func ~window (region: IntBox2D, imageX, imageY: FloatImage) {
```
Comment (emilwestergren) [inline source/draw/RasterMonochrome.ooc]: So you removed the unoptimized and made the optimized default?

Comment (tomasmahlberg) [inline source/draw/RasterMonochrome.ooc]: Default and the only one, yes.

**Comment by marcusnaslund** [comment]
Conflicts.

**Comment by tomasmahlberg** [comment]
Updated.

**Pull request merged**